README.md

```diff
@@ -356,5 +356,5 @@ The plugin, when applied, automatically creates tasks for the module (all module
 - `koverXmlReport` - Generates code coverage XML report for all module's test tasks.
 - `koverHtmlReport` - Generates code coverage HTML report for all module's test tasks.
 - `koverReport` - Executes both `koverXmlReport` and `koverHtmlReport` tasks.
-- `koverCollectReports` - Collects reports from all submodules in one directory. Main directory is `$buildDir/reports/kover/all`, names for XML reports and dirs for HTML is a projects names.
+- `koverCollectReports` - Collects reports from all submodules in one directory. Main directory is `$buildDir/reports/kover/all`, names for XML reports and dirs for HTML is a projects names. This task does not clear the output directory, but only copies the reports, and the files in it can be overwritten. It is most reliable to call this task only once after `clean` task.
```

Comment:
This description still does not state whether actual `test` and `koverReport` tasks are invoked or not. 

Also, I don't understand the phrase 'It is most reliable to call this task only once after `clean` task.' What's wrong with invoking `koverCollectReports` after e.g. changing tests? Overwriting old reports seems OK to me. If tests were changed, I get new coverage stats, if they were not, old file still provides correct data.

Reply:
Then we probably should clean a particular html directory if we know that we will overwrite the report. Saying 'always invoke clean before using this task' is not a very great user experience. No one will do that anyway, and for big projects consequences of clean may be big (imagine we need to invoke `clean` on Kotlin compiler repo deleting all the artifacts just to re-collect coverage)

Reply:
Wouldn't something like this:

```
fun collect() {
... 
     htmlDirs.forEach { (p, d) ->
            outputDir.dir("html/$p").deleteIfExists()
            project.copy {
                it.from(d)
                it.into(outputDir.dir("html/$p"))
            }
     }
```

solve the problem?